contents.json

```diff
@@ -18344,11 +18344,11 @@
         "swift"
       ],
       "screenshots": [
-        "https://i.imgur.com/g3vPueY.png"
+        "https://pgpro.app/screenshots/current/"
```

Comment:
Thanks for the update @lucanaef - however these need to link to an image or images

Could you update this to 
  `"screenshots": [ "http ... image1.png" ]` or
  `"screenshots": [ "http ... image1.png", "http ... image2.png" ]`